src/Network/Network/VirtualRouter/GetAzureRMVirtualRouterCommand.cs

```diff
@@ -55,7 +55,7 @@ public partial class GetAzureRmVirtualRouter : VirtualRouterBaseCmdlet
             HelpMessage = "ResourceId of the virtual router.",
             ValueFromPipelineByPropertyName = true)]
         [ValidateNotNullOrEmpty]
-        [ResourceIdCompleter("Microsoft.Network/virtualRouters")]
+        [ResourceIdCompleter("Microsoft.Network/virtualHubs")]
```

Comment:
This resourceId is the Id of that target virtual router to be retrieved I suppose, description should not be changed